README.md

```diff
@@ -12,42 +12,19 @@ These tests can all be run at a go using the "run" file or executed via a Docker
 
 ###General Configuration###
 
+Python 3.5.2 or greater is required.
+
 It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
 in order to have an isolated environment.
 
-From /kinto/
+From this directory
 
-1. virtualenv .
+1. virtualenv . -p /path/to/python
 2. ./bin/pip install -r dev-requirements.txt
 
-NOTE -- as of 01/14/2016 we are relying on a specific branch of the [Kinto Python client](https://github.com/Kinto/kinto.py)
-that implements the replication features. When the pull request containing that
-code gets merged into the master branch the dev-requirements.txt file will be
-updated.
 
 #Run Tests#
```

Comment:
it looked to me like you had three test directories (each with a different test type).  Are they perhaps all config tests?  (I noticed you droped the section below so i wasn't sure)